website/docs/security/tls.md

```diff
@@ -0,0 +1,124 @@
+---
+sidebar_position: 2
+description: TLS
```

Comment:
Great job on the documentation! Here we can include a more descriptive information here. Something like: `Learn to secure connections using TLS`

Reply:
Replace it :+1: